src/clients/api/queries/getVTokenCash.ts

```diff
@@ -0,0 +1,17 @@
+import BigNumber from 'bignumber.js';
+import { VBep20, VBnbToken } from 'types/contracts';
+
+export interface IGetVTokenCashInput {
+  vTokenContract: VBep20 | VBnbToken;
+}
+
+export type GetVTokenCashOutput = BigNumber;
+
+const getVTokenCash = async ({
+  vTokenContract,
+}: IGetVTokenCashInput): Promise<GetVTokenCashOutput> => {
+  const res = await vTokenContract.methods.getCash().call();
```

Comment:
This will need to be wrapped in our error new handling to check for the failure event

Reply:
That's a view function, there's no transaction receipt returned